### PR TITLE
Additition of the new constants

### PR DESCRIPTION
**Pull request by OlehKSS**
Check out this additional constants for  the api. Is naming ok?
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23issuecomment-171676978%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23discussion_r49779759%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23discussion_r49797236%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23issuecomment-172013930%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23discussion_r49901427%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23discussion_r49901716%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23issuecomment-172087421%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23issuecomment-172093807%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23issuecomment-172094730%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23issuecomment-172309744%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23issuecomment-171676978%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%20am%20sorry%20for%20misspelling%20addition...%22%2C%20%22created_at%22%3A%20%222016-01-14T15%3A40%3A54Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/16047980%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/OlehKSS%22%7D%7D%2C%20%7B%22body%22%3A%20%22Maybe%20it%20will%20make%20sence.%22%2C%20%22created_at%22%3A%20%222016-01-15T16%3A52%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/16047980%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/OlehKSS%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20suppose%20that%20all%20this%20because%20of%20merge%20with%20the%20latest%20%20master.%22%2C%20%22created_at%22%3A%20%222016-01-15T20%3A41%3A52Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/16047980%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/OlehKSS%22%7D%7D%2C%20%7B%22body%22%3A%20%22How%20can%20I%20change%20this%20erorr%3F%20%22%2C%20%22created_at%22%3A%20%222016-01-15T21%3A12%3A57Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/16047980%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/OlehKSS%22%7D%7D%2C%20%7B%22body%22%3A%20%22Or%20maybe%20it%20will%20be%20easier%20to%20merge%20it%20with%20my%20api%20branch.%20But%20in%20that%20case%20it%20will%20take%20some%20time%2C%20and%20lately%20we%20will%20have%20to%20spend%20some%20time%20in%20order%20to%20syncronize%20our%20services.%20Will%20it%3F%22%2C%20%22created_at%22%3A%20%222016-01-15T21%3A17%3A21Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/16047980%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/OlehKSS%22%7D%7D%2C%20%7B%22body%22%3A%20%22added%20separate%20pull%20request%20%23140.%20Please%20check%20and%20I%20will%20merge%22%2C%20%22created_at%22%3A%20%222016-01-17T10%3A15%3A39Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7273003%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20be455be9896e4c5cbb7b49ee74cfb71439fd9705%20TCMSApp/src/client/app/blocks/fakers/fake-defects.factory.js%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23discussion_r49901716%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22what%20have%20you%20changed%20in%20fake%20defects%20factory%3F%22%2C%20%22created_at%22%3A%20%222016-01-15T20%3A33%3A04Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7273003%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20TCMSApp/src/client/app/blocks/fakers/fake-defects.factory.js%3AL1-79%22%7D%2C%20%22Pull%200c166da8e0d1144cdf65f5adec7cb2a8d584fc0f%20TCMSApp/src/client/app/core/constants.js%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23discussion_r49779759%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%20think%20would%20be%20better%20like%20this%3A%5Cr%5Cn.constant%28%27apiURL%27%2C%20apiURL%29%22%2C%20%22created_at%22%3A%20%222016-01-14T20%3A21%3A08Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/11821757%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/vp9920%22%7D%7D%2C%20%7B%22body%22%3A%20%22yep%22%2C%20%22created_at%22%3A%20%222016-01-14T22%3A44%3A05Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7273003%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20TCMSApp/src/client/app/core/constants.js%3AL7-19%22%7D%2C%20%22Pull%20be455be9896e4c5cbb7b49ee74cfb71439fd9705%20TCMSApp/src/client/app/core/config.js%2017%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-012/pull/126%23discussion_r49901427%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22why%20there%20are%20new%20blank%20lines%20and%20%60timeOut%3A%201000%60%20removed%3F%22%2C%20%22created_at%22%3A%20%222016-01-15T20%3A30%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7273003%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20TCMSApp/src/client/app/core/config.js%3AL30-40%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-012/pull/126#issuecomment-171676978'>General Comment</a></b>
- <a href='https://github.com/OlehKSS'></a> I am sorry for misspelling addition...
- <a href='https://github.com/OlehKSS'></a> Maybe it will make sence.
- <a href='https://github.com/OlehKSS'></a> I suppose that all this because of merge with the latest  master.
- <a href='https://github.com/OlehKSS'></a> How can I change this erorr?
- <a href='https://github.com/OlehKSS'></a> Or maybe it will be easier to merge it with my api branch. But in that case it will take some time, and lately we will have to spend some time in order to syncronize our services. Will it?
- <a href='https://github.com/ITsvetkoFF'></a> added separate pull request #140. Please check and I will merge
- [ ] <a href='#crh-comment-Pull 0c166da8e0d1144cdf65f5adec7cb2a8d584fc0f TCMSApp/src/client/app/core/constants.js 4'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-012/pull/126#discussion_r49779759'>File: TCMSApp/src/client/app/core/constants.js:L7-19</a></b>
- <a href='https://github.com/vp9920'></a> I think would be better like this:
.constant('apiURL', apiURL)
- <a href='https://github.com/ITsvetkoFF'></a> yep
- [ ] <a href='#crh-comment-Pull be455be9896e4c5cbb7b49ee74cfb71439fd9705 TCMSApp/src/client/app/core/config.js 17'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-012/pull/126#discussion_r49901427'>File: TCMSApp/src/client/app/core/config.js:L30-40</a></b>
- <a href='https://github.com/ITsvetkoFF'></a> why there are new blank lines and `timeOut: 1000` removed?
- [ ] <a href='#crh-comment-Pull be455be9896e4c5cbb7b49ee74cfb71439fd9705 TCMSApp/src/client/app/blocks/fakers/fake-defects.factory.js 1'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-012/pull/126#discussion_r49901716'>File: TCMSApp/src/client/app/blocks/fakers/fake-defects.factory.js:L1-79</a></b>
- <a href='https://github.com/ITsvetkoFF'></a> what have you changed in fake defects factory?

<a href='https://www.codereviewhub.com/ITsvetkoFF/Kv-012/pull/126?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/ITsvetkoFF/Kv-012/pull/126?approve=1'></a>&nbsp;<a href='https://github.com/ITsvetkoFF/Kv-012/pull/126'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by ITsvetkoFF** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/ITsvetkoFF/Kv-012/pull/126'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by OlehKSS** [comment]
I am sorry for misspelling addition...

File: TCMSApp/src/client/app/core/constants.js [hunk 1]
```diff
@@ -7,4 +7,12 @@
         .constant('moment', moment)
         .constant('faker', faker)
         .constant('Trello', Trello)
+        .constant('apiURL', {
```
Comment (vp9920) [inline TCMSApp/src/client/app/core/constants.js]: I think would be better like this:
.constant('apiURL', apiURL)

Comment (ITsvetkoFF) [inline TCMSApp/src/client/app/core/constants.js]: yep

**Comment by OlehKSS** [comment]
Maybe it will make sence.

File: TCMSApp/src/client/app/core/config.js [hunk 2]
```diff
@@ -28,8 +30,10 @@
 
         // toastr configuration
         angular.extend(toastrConfig, {
-            positionClass: 'toast-bottom-right',
-            timeOut: 1000
+            positionClass: 'toast-bottom-right'
         });
+
```
Comment (ITsvetkoFF) [inline TCMSApp/src/client/app/core/config.js]: why there are new blank lines and `timeOut: 1000` removed?

File: TCMSApp/src/client/app/blocks/fakers/fake-defects.factory.js [hunk 3]
```diff
@@ -1,67 +1,78 @@
+/**
```
Comment (ITsvetkoFF) [inline TCMSApp/src/client/app/blocks/fakers/fake-defects.factory.js]: what have you changed in fake defects factory?

**Comment by OlehKSS** [comment]
I suppose that all this because of merge with the latest  master.

**Comment by OlehKSS** [comment]
How can I change this erorr?

**Comment by OlehKSS** [comment]
Or maybe it will be easier to merge it with my api branch. But in that case it will take some time, and lately we will have to spend some time in order to syncronize our services. Will it?

**Comment by ITsvetkoFF** [comment]
added separate pull request #140. Please check and I will merge

**Pull request closed without merging**